docs/octopus-cloud/Uptime SLO

```diff
@@ -0,0 +1,19 @@
+---
+title: Octopus Cloud Uptime SLO
+position: 50
+description: The uptime SLO for Octopus Cloud instances
+---
+
+## Octopus Cloud Uptime
```

Comment:
I don't think a H2 heading is required as there is no preceding paragraph. This results in a H1 stating **Octopus Cloud Uptime SLO**, directly followed by a H2 with **Octopus Cloud Uptime**. An alternative here would be to add a preceding paragraph stating what the page is for.
```suggestion
```